files/matreshka/update.json

```diff
@@ -3,6 +3,13 @@
   "name": "matreshka",
   "repo": "matreshkajs/matreshka",
   "files": {
-    "include": ["matreshka.min.js", "matreshka.js", "matreshka.min.map"]
+    "include": [
+		"matreshka.min.js",
+		"matreshka.js",
+		"matreshka.min.map",
+		"magic/matreshka-magic.min.js",
```

Comment:
If you want the folder to be created you need to use a different format
`./magic/matreshka-magic.min.js`
`./` retains the directory structure, otherwise all folders are stripped